app/templates/_service_attributes.html

```diff
@@ -1,17 +1,21 @@
 {% import "toolkit/summary-table.html" as summary %}
-{% for attributeGrouping in service.attributes %}
-  {{ summary.heading(attributeGrouping.name) }}
-  {% call(item) summary.list_table(
-    attributeGrouping.rows,
-    caption=attributeGrouping.name,
-    field_headings=[
-      "Name", "Content",
-    ],
-    field_headings_visible = False
-  ) %}
-    {% call summary.row() %}
-      {{ summary.field_name(item.label) }}
-      {{ summary[item.type](item.value, item.assurance) }}
+
+{% for section in service.summary_manifest %}
+  {% if not section.is_empty %}
+    {{ summary.heading(section.name, id=section.slug) }}
+
+    {% call(question) summary.list_table(
+      section.questions,
+      caption=section.name,
+      field_headings_visible=False
+    ) %}
+      {% if not question.is_empty %}
+        {% call summary.row() %}
+          {{ summary.field_name(question.label, bold=question.followup) }}
+          {{ summary[question.type](question.value, question.assurance, bold=question.followup) }}
```

Comment:
So there's a bit of extra logic required here: only the first question in a dependency chain should be bold; any subsequent followups within the same chain should *not* be bold.